integration/basic_test.go

```diff
@@ -629,3 +629,37 @@ func TestDebugError(t *testing.T) {
 		require.NoError(t, db.Client().Ping(ctx, nil), "other errors should not close connection")
 	})
 }
+
+func TestPing(t *testing.T) {
+	// Shouldn't be ran in parallel, as NonExistentDB test requires
+	// the current state of database list, which can be changed by other tests.
+
+	ctx, collection := setup.Setup(t)
+	db := collection.Database()
```

Comment:
What is the expected response if ping fails, for example if DB is not accepting connection.

Reply:
I believe the expected results are different for each backend.

MongoDB doesn't have any additional backend so they probably return `{ok:1}` or the request is just not delivered to the service. 

In our case - apart from "client -> FerretDB" connection we also want to make sure that FerretDB has connection with selected backend.

As different backend implementation might return different errors I guess the error message should be backend dependent. WDYT?

Reply:
As long as it does not include `ok: 1`, we are good, but we should check that

Reply:
Thanks for clarification :+1:

As a user of ping, if ping doesn't return `{ok :1}` I would expect there is some issue with connection or setup. Can this be tested?

Reply:
Not really, I'm not sure how can we simulate the lack of connection with SQLite.

Reply:
For example, by using an existing non-writable directory

Reply:
We probably won't have any error here. Should ping check for that?

Reply:
> We probably won't have any error here

Why not try that?

Reply:
Do we even have any Setup that changes sqlite dbs location?

Reply:
Yes, but it's a constant location at the moment https://github.com/FerretDB/FerretDB/blob/main/integration/setup/startup.go#L50

Reply:
This variable is used by newHandlerFunc, and setup functions connect to already existing handlers.

Reply:
Creating the handler is part of the test setup too. What is the challenge here? :thinking:

Reply:
Creating the handler is part of Setup function, which is called only once from `main_test.go`. From my understanding there's no way of easily modifying the SQLite db directory for one specific test/test case.

Reply:
I did quick experiment and found out even if we have 0x000 permission, it's not possible for query to fail, so ping will not fail on this.
https://github.com/FerretDB/FerretDB/pull/2997